controller/Readme.md

```diff
@@ -14,9 +14,99 @@ PlayOS controller is an OCaml application that manages various system tasks for
 - `server/`: main application code
 - `bin/`: binaries to start a dev server
 
+## Prerequisites
+Since PlayOS uses `Connman` for managing networks, it must be configured on your system.
+Replace your NetworkManager config with the following (remember to fill in `<your-host-name>`):
```

Comment:
I have a much shorter config on my own machine (I use connman regularly/permanently), was this one necessary for you to use in full?

Maybe also good to note that this assumes NixOS, while it should be possible to build PlayOS on another Linux distro using just nix.

Reply:
I don't remember the details of why my config is so long, but it very well might contain unnecessary things. Could you share yours?

[Add note about assuming NixOS to Connman setup guidance](https://github.com/dividat/playos/pull/136/commits/5fd37cbfa209f3621d54096e7760af69b45adcad)

Reply:
I think this should be enough (it is, I can use the controller like this):

```
  networking = {
    hostName = "foo";
    wireless.enable = true;
    networkmanager.enable = false;
  };
  
  services.connman = {
    enable = true;
    networkInterfaceBlacklist = [ "vboxnet" "zt" ];
  };
```

Reply:
https://github.com/dividat/playos/pull/136/commits/202d5eda1f640bc3697aaf6f218866338be714cf